System/macosnightshift.1h.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" bash
```

This will allow tools installed by Homebrew to be used without specifying the full path. Homebrew on M1 Macs install into a different directory. This will catch both of them.